server_admin/topics/threat/brute-force.adoc

```diff
@@ -10,13 +10,39 @@ go to the `Brute Force Detection` sub-tab.
 .Brute Force Detection
 image:{project_images}/brute-force.png[]
 
-The way this works is that if there are `Max Login Failures` during a period of `Failure Reset Time`,
-the account is temporarily disabled for the `Wait Increment` multiplied by the number of failures over the max.  After
-`Failure Reset Time` is reached all failures are wiped clean.  The `Max Wait` is the maximum amount of time
-an account can be disabled.  Another preventive measure is that if there are subsequent login failures for one
-account that are too quick for a human to initiate the account will be disabled.  This is controlled by the
-`Quick Login Check Milli Seconds` value.  So, if there are two login failures for the same account within that value,
-the account will be disabled for `Minimum Quick Login Wait`.
+There are 2 different configurations for brute force detection; permanent lockout and temporary lockout. Permanent lockout will disable a users account after an attack is detected; the account will be disabled until an administrator renables it. Temporary lockout will disable a users account for a time period after an attack is detected; the time period for which the account is disabled increases the longer the attack continues. The algorithms for each configuration are described below:
```

Comment:
"The algorithms _and their parameteris_ for each configuration are described below:" should be in separate paragraphs.